src/CMakeLists.txt

```diff
@@ -1,13 +1,25 @@
 add_executable(timg)
-target_sources(timg PRIVATE image-display.cc terminal-canvas.cc timg.cc)
-target_link_libraries(timg PkgConfig::GRAPHICSMAGICKXX)
+target_sources(timg PRIVATE
+  image-display.cc
+  image-source.cc
+  renderer.cc
+  terminal-canvas.cc
+  timg.cc
+  video-display.cc
+  display-options.h
+  image-source.h
+  terminal-canvas.h
+  timg-time.h
+  image-display.h
+  renderer.h
+  thread-pool.h
+  video-display.h
+)
 
+target_link_libraries(timg PkgConfig::GRAPHICSMAGICKXX Threads::Threads)
 target_include_directories(timg PRIVATE ${CMAKE_BINARY_DIR}/src)
 
-# instead of just telling cmake "c++11, please", this a more modern approach:
-# https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html
-# https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/ "Setting the C++ standard based on features"
```

Comment:
The cmake page linked abe actually said that my knowledge of this cxx features is already outdated: 
`"These individual features are now less relevant and projects should generally prefer to use the high level meta features instead. Individual compile features are not provided for C++ 17 or later."
`
So I did just specify language level…